RespawnTimer/API/Features/ExternalTeams/RRH.cs

```diff
@@ -0,0 +1,19 @@
+#if EXILED
+namespace RespawnTimer.API.Features.ExternalTeams
+{
+    using System;
+    using System.Reflection;
+
+    public class SerpentsHandTeam : ExternalTeamChecker
+    {
+        public override void Init(Assembly assembly)
+        {
+            PluginEnabled = true;
+            
+            Type mainClass = assembly.GetType("RRR.Alpha-1_Red-Right-Hand");
```

Comment:
Alpha-1

Reply:
But nothing would change on my end with RRH

Reply:
> Alpha-1

That is the name of the plugin